packages/shared/providers.yaml

```diff
@@ -1369,6 +1369,16 @@ teamwork:
     token_url: https://www.teamwork.com/launchpad/v1/token.json
     proxy:
         base_url: https://api.surveymonkey.com
+tiktok-ads:
+    auth_mode: OAUTH2
+    authorization_url: https://business-api.tiktok.com/portal/auth
+    token_url: https://business-api.tiktok.com/open_api/v1.3/oauth2/access_token/
+    proxy:
+        base_url: https://business-api.tiktok.com/open_api/v1.3/
+    token_params:
+        grant_type: authorization_code
+    authorization_url_replacements:
```

Comment:
I saw we have a schema.json but it seems outdate, we don't use it for validation?